### PR TITLE
Fix location_event_counts not working with subsets of event types

### LINKED ISSUE #1015: EventTypes custom field needs to deserialise with schema
The `EventTypes` custom field validates as a list containing any combination of the events tables, and hence gets used to populate the `table`/`tables` argument. This won't actually work though, because `table`/`tables` are expected to be schema-qualified with `events.TABLE_NAME`.

So, until the machinery inside is changed to be a bit more sensible - this needs to deserialise with the `events.` prefix.

### PR DESCRIPTION
**Pull request by greenape**
Closes #1015

### I have:

- [x] Formatted any Python files with [black](https://github.com/ambv/black)
- [x] Brought the branch up to date with master
- [x] Added any relevant Github labels
- [x] Added tests for any new additions
- [ ] Added or updated any relevant documentation
- [ ] Added an Architectural Decision Record (ADR), if appropriate
- [ ] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate
- [x] Updated the [Changelog](https://github.com/Flowminder/FlowKit/blob/master/CHANGELOG.md) 

### Description

Added a custom deserialiser to `EventTypes` field which dedupes the list, and appends the `events.` prefix required to use this as a `table` argument.

### CODE REVIEW

File: flowmachine/flowmachine/core/server/query_schemas/custom_fields.py [hunk 1]
```diff
@@ -11,7 +11,9 @@
 
 class EventTypes(fields.List):
     """
-    A string representing an event type, for example "calls", "sms", "mds", "topups".
+    A list of strings representing an event type, for example "calls", "sms", "mds", "topups".
+
+    When deserialised, will be uniqueified, and prefixed with "events."
```
Comment (maxalbert) [inline flowmachine/flowmachine/core/server/query_schemas/custom_fields.py]: What do you mean by "uniqueified"?

(And where is this actually happening - in the call to `super()`?)

Comment (greenape) [inline flowmachine/flowmachine/core/server/query_schemas/custom_fields.py]:
```suggestion
    When deserialised, will be deduped, and prefixed with "events."
```

File: flowmachine/flowmachine/core/server/query_schemas/custom_fields.py [hunk 2]
```diff
@@ -29,6 +31,12 @@ def __init__(self, required=False, validate=None, **kwargs):
             **kwargs,
         )
 
+    def _deserialize(self, value, attr, data, **kwargs):
```
Comment (maxalbert) [inline flowmachine/flowmachine/core/server/query_schemas/custom_fields.py]: Can we stick a comment in here to say that this is a temporary workaround until the underlying issue mentioned in #1015 is solved and then we can hopefully remove this again?

Comment (greenape) [inline flowmachine/flowmachine/core/server/query_schemas/custom_fields.py]:
```suggestion
    def _deserialize(self, value, attr, data, **kwargs):
        # Temporary workaround for https://github.com/Flowminder/FlowKit/issues/1015 until underlying issue resolved
```

**Pull request merged**